src/Entity/Cover.php

```diff
@@ -36,25 +36,14 @@
  *             "controller"=CreateCoverAction::class,
  *             "deserialize"=false,
  *             "validation_groups"={"Default", "cover_create"},
- *             "swagger_context"={
- *                 "consumes"={
- *                     "multipart/form-data",
- *                 },
- *                 "parameters"={
- *                     {
- *                         "in"="formData",
- *                         "name"="cover",
- *                         "type"="file",
- *                         "description"="The cover (image file) to upload",
- *                     },
- *                 },
+ *             "openapi_context"={
  *                 "requestBody"={
  *                     "content"={
  *                         "multipart/form-data"={
  *                             "schema"={
  *                                 "type"="object",
  *                                 "properties"={
- *                                     "file"={
+ *                                     "cover"={
```

Comment:
Changed field name from "file" to "cover"? Will that not change the client?

Reply:
We look for `cover` in the `CreateCoverAction`:
```
$uploadedFile = $request->files->get('cover');
if (!$uploadedFile) {
    throw new BadRequestHttpException('"file" is required');
}
```
After I updated from Swagger to OpenAPi the client started posting `file` causing a `BadRequestHttpException('"file" is required')`

Reply:
Just need to test that with the php-client to ensure that it still works as expected.